DEVELOPING.md

```diff
@@ -114,6 +114,14 @@ are at:
 A build for a specific commit may not always be available: for instance, if the build was cancelled early or failed to
 complete.
 
+AppVeyor builds
+---------------
+Independently from Amazon S3 hosting service, AppVeyor stores its Windows builds temporarily for 6 months. In contrast to the Amazon S3 hosting, these builds are available for _all_ pull requests. The URL to these builds includes AppVeyor's job number and is unrelated to commit SHA, e.g.
+
+* https://ci.appveyor.com/api/buildjobs/76sh4cu20rh50bp1/artifacts/art_folder.zip
```

Comment:
thanks! tiny formatting nitpick -- this should be a code block rather than a single bullet point

Reply:
Thanks @snappizz 
If I change it to be a code block, it stops being a URL. Is that OK?

Reply:
yeah, it's just for display purposes to demonstrate what the URL looks like, right?